client/src/commands/UserInputUtil.ts

```diff
@@ -199,6 +199,12 @@ export class UserInputUtil {
             });
         }
 
+        // If there are no installed smart contract packages
+        if (quickPickItems.length === 0) {
+            vscode.window.showErrorMessage('Packages must be installed before they can be instantiated');
```

Comment:
I think this error message could be improved... something like `There are currently no smart contracts installed on any peers that this channel is joined to. Please install a smart contract before instantiating`

Reply:
My preference: "No smart contracts are installed on peers in this channel. Install a smart contract before instantiating."